Data/Entities/PageDefinitionDO.cs

```diff
@@ -44,5 +44,15 @@ public string UriString
 
         public string Description { get; set; }
         public string PageName { get; set; }
+
+        [NotMapped]
+        public List<string> PlanTemplatesIds { get; set; }
+
+        [Column("PlanTemplatesIds")]
+        public string PlanTemplatesString
+        {
+            get { return PlanTemplatesIds != null && PlanTemplatesIds.Any() ? string.Join(",", PlanTemplatesIds.Distinct()) : string.Empty; }
+            set { PlanTemplatesIds = value?.Split(',').ToList(); }
```

Comment:
I realise that PlanTemplates are stored in MT-database, and PlanTemplate.Id is a dynamic field inside PlanTemplateCM entry.
I understand why we serialize a list of IDs to a single string.
However, in other places we don't use custom serializers, and use standard JSON.Net serializers instead.
My suggestion is: let's use same JSON.Net approach here, so we could have more interopability with the rest of code.